CONTRIBUTING.rst

```diff
@@ -46,6 +47,7 @@ Or Ubuntu with:
 
 .. code:: bash
 
+    sudo apt-get install unixodbc unixodbc-dev
```

Comment:
See line 55. Also, I'm pretty sure that `unixodbc` is a prerequisite of `unixodbc-dev` because development packages tend to depend on the non-development ones.

Reply:
Removed `unixodbc`. Based on our `.travis.yml` file, I think that only `unixodbc-dev` is required.